source/_integrations/pioneer.markdown

```diff
@@ -43,6 +43,16 @@ sources:
   required: false
   default: Empty list (i.e., no source selection will be possible)
   type: list
+alternative_volume_control:
+  description: Whether an alternative method to set volume to a specific amount should be used (enable this if you have problems with said functionality)
+  required: false
+  default: false
+  type: boolean
+max_volume:
+  description: Maximum volume reported by the receiver (can be used in conjunction with `alternative_volume_control` to fix volume issues)
+  required: false
+  default: 185
+  type: integer
```

Comment:
We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>